client/src/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+  //because I'm not using state, with more time I would probably restyle this to a functional component
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick() {
+    let sheets = document.styleSheets;
+    let sheet;
+    let newCSS = document.querySelector('#css').value;
+    if (sheets.length === 0) {
+      sheet = document.createElement('style');
+      sheet.innerHTML = newCSS;
+      document.body.appendChild(sheet);
+    } else {
+      sheet = sheets[0];
+      sheet.insertRule(newCSS, sheet.cssRules.length);
+    }
+  }
+
+  render() {
+    return (
+      <div>

```

Comment:
All the dummy markup can go into a separate component responsible for rendering dummy HTML. Technically it's not App's job to take care of that.